packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/NoOpVoidResult.java

```diff
@@ -0,0 +1,16 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera;
+
+import androidx.annotation.NonNull;
+
+/** A convenience class for results of a Pigeon Flutter API method call that perform no action. */
```

Comment:
Please add [this line](https://github.com/flutter/packages/blob/17b714a361460ac51b3dd51a1bbf0dec26fdbe3c/packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/NoOpVoidResult.java#L14) to the comment as well; it's true of this plugin as well.